ingestion_tools/scripts/tests/conftest.py

```diff
@@ -63,3 +63,9 @@ def s3_client(endpoint_url: str) -> S3Client:
         endpoint_url=endpoint_url,
         config=Config(signature_version="s3v4"),
     )
+
+
+@pytest.fixture
+def local_test_data_dir() -> str:
+    cd = os.path.dirname(__file__).split("/")[0:-3]
+    return os.path.join("/", *cd, "test_infra/test_files")
```

Comment:
Reading files from moto3 was causing my pycharm debug environment to hang. Reading from the local files system let me get around this. This is the same reason why i added the test_infra directory as a volume do the docker image since i'm running the tests in docker.